packages/interface/src/GraphQLLanguageService.js

```diff
@@ -46,6 +46,8 @@ import {
 
 import {parse, print} from 'graphql';
 import {getAutocompleteSuggestions} from './getAutocompleteSuggestions';
+import {getHoverInformation} from './getHoverInformation';
+import type {Hover} from 'vscode-languageserver-types';
```

Comment:
I don't have a good lint rule set up for this, but I'd encourage including type imports on the top of the list of imports.

Reply:
Done